src/dwarf_parser.h

```diff
@@ -70,6 +71,11 @@ class Dwarf {
     return nullptr;
   }
 
+  uint64_t get_prologue_size(const std::string &function __attribute((unused)))
```

Comment:
JC, why is this needed?

Reply:
This just need a cleanup from the first version of this PR which only skipped the prologue.